aten/src/ATen/core/alias_info.h

```diff
@@ -98,11 +98,22 @@ inline std::ostream& operator<<(std::ostream& out, const AliasInfo& aliasInfo) {
     }
     out << set.toUnqualString();
   }
-  out << ")";
-
-  if (!aliasInfo.containedTypes().empty()) {
```

Comment:
not sure whether removing it makes debugging harder or not. We can also put a separate method to print to string, but @suo is best to decide